pkg/controllers/multiclusterservice/endpointslice_dispatch_controller.go

```diff
@@ -338,11 +338,18 @@ func (c *EndpointsliceDispatchController) dispatchEndpointSlice(ctx context.Cont
 				"Consumer cluster %s is not ready, skip to propagate EndpointSlice", clusterName)
 			continue
 		}
-		if !helper.IsAPIEnabled(clusterObj.Status.APIEnablements, util.EndpointSliceGVK.GroupVersion().String(), util.EndpointSliceGVK.Kind) {
```

Comment:
The `EndpointSlice, discovery.k8s.io/v1` API is kind of a hard restriction for the MultiClusterService feature. 

I'm thinking if it is appropriate to check this here.   Any thoughts?